modules/p2/ding_sharer/ding_sharer.module

```diff
@@ -95,21 +96,28 @@ function template_preprocess_ding_sharer(&$variables) {
  */
 function theme_ding_sharer($variables) {
   $out = '';
+  $anchor_class = '';
 
-  if (!empty($variables['label'])) {
+  if (!empty($variables['label']) && $variables['type'] == 'normal') {
     $out .= '<div class="label">' . $variables['label'] . '</div>';
   }
 
   $out .= '<div class="share-buttons">';
   foreach ($variables['communities'] as $community => $text) {
-    $out .= l(
-      $text,
-      '#',
-      array(
-        'attributes' => array('class' => array($community . '-share')),
-        'external' => TRUE,
-      )
-    );
+    if ($text !== '0') {
```

Comment:
Again this comparison with the string `0`. Please avoid this by irrelevant communities beforehand.

Reply:
:+1: